gp-templates/glossary-view.php

```diff
@@ -18,10 +19,10 @@
 gp_tmpl_header();
 
 /* translators: 1: Locale english name. 2: Project name. */
-$glossary_title = __( 'Glossary for %1$s translation of %2$s', 'glotpress' );
+$glossary_title = __( 'Project Glossary for %1$s translation of %2$s', 'glotpress' );
```

Comment:
Not sure if the prefix is helpful here since the title already includes the project name. Should we remove it to keep the title short?

Reply:
I agree, it is unnecessary.